src/components/EditCollapse.js

```diff
@@ -216,6 +219,22 @@ const EditCollapse = ({ ...props }) => {
               </InputGroupAddon>
             </InputGroup>
           </Form>
+          <Form inline className="mb-2 mr-2">
+            <InputGroup>
+              <InputGroupAddon addonType="prepend">
+                <InputGroupText>
+                  <Input
+                    addon
+                    type="checkbox"
+                    aria-label="Checkbox for following text input"
+                    checked={specifyEdition}
+                    onChange={() => setSpecifyEdition(!specifyEdition)}
+                  />
+                </InputGroupText>
+              </InputGroupAddon>
+              <Input disabled value="Specify Versions" />
```

Comment:
I’m not familiar with what this line, Input component with a disabled value string, does.

Reply:
Usually you use this to dynamically disable inputs but I just like the styling of this component so I'm using an input as a label. I won't defend it as best practice but I think it's fine here